### PR TITLE
Add a proper timeout before mssql server starts

### PR DESCRIPTION
**Pull request by elkbullwinkle**
Current static timeout is not sufficient on every system and causes some unexpected errors. This one will have a proper timeout up to 90 secs, which can be extended.

### CODE REVIEW

File: mssql/custom/setup.sh [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
+  sleep 2
+done
 
-echo "Running initial SQL statements"
+echo "Saying adios to CLR strict security"
 
-# run the setup script to create the DB and the schema in the DB
-/opt/mssql-tools/bin/sqlcmd -S localhost -U SA -P "$SA_PASSWORD" -d master -i /custom/setup.sql
\ No newline at end of file
+#Goodbye CLR strict security
```
Comment (derschatta) [inline mssql/custom/setup.sh]: how is this change related to the timeout?

Comment (elkbullwinkle) [reply mssql/custom/setup.sh]: It didn't do what it said it does - "# run the setup script to create the DB and the schema in the DB", that's a copy paste from a script that I had for a while

Comment (derschatta) [reply mssql/custom/setup.sh]: yes the comment is wrong but it runs the same commands you pass here in your change.
https://github.com/totara/totara-docker-dev/blob/master/mssql/custom/setup.sql
Anyway, I think this pr should only change the timeout

Comment (derschatta) [reply mssql/custom/setup.sh]: `echo "Running initial SQL statements"`
in combination with running /custom/setup.sql is still true as you could add more initial sql statements in there

Comment (elkbullwinkle) [reply mssql/custom/setup.sh]: Whatever. There was also double fetching repo key. Fix it yourself.
